packages/js/src/TelnyxRTC.ts

```diff
@@ -126,4 +130,58 @@ export default class TelnyxRTC extends TelnyxRTCClient {
   newCall(options: ICallOptions) {
     return super.newCall(options);
   }
+
+  /**
+   * Checks if the running browser has support for TelnyRTC
+   *
+   * @return an object with telnyx webrtc information.
+   *
+   * @examples
+   *
+   * If your browser has supported
+   *
+   * ```js
+   * const info = TelnyxRTC.webRTCInfo();
+   * console.log(info) // => 'This browser does not support @telnyx/webrtc. \nPlease, use Chrome|Firefox|Safari.'
```

Comment:
It looks like the error example got switched
```suggestion
   * const isWebRTCSupported = info.supportWebRTC;
   * console.log(isWebRTCSupported); // => true
```